cloud_governance/main/environment_variables.py

```diff
@@ -195,6 +195,7 @@ def __init__(self):
         self._environment_variables_dict['DEFAULT_ADMINS'] = literal_eval(EnvironmentVariables.get_env('DEFAULT_ADMINS', '[]'))
         self._environment_variables_dict['KERBEROS_USERS'] = literal_eval(EnvironmentVariables.get_env('KERBEROS_USERS', '[]'))
         self._environment_variables_dict['POLICIES_TO_ALERT'] = literal_eval(EnvironmentVariables.get_env('POLICIES_TO_ALERT', '[]'))
+        self._environment_variables_dict['ONLY_ADMINS'] = EnvironmentVariables.get_boolean_from_environment('ONLY_ADMINS', False)
```

Comment:
Do we need admin email list ?

Reply:
No, 
I added the extra var EMAIL_TO here we can add the admin mail and EMIAL_CC to add other users

Reply:
Pls fix it as we discussed, **ADMINS_ONLY** (instead of ONLY_ADMINS) should be list of emails, if the list is filled, need to send the email TO them